app/src/main/kotlin/com/example/android/picover/presentation/ProfileDrawerContent.kt

```diff
@@ -1,14 +1,19 @@
 package com.example.android.picover.presentation
```

Comment:
@marcinWisnia Package should be corrected now

Reply:
good catch :) thanks, will be fixed in upcoming PR :)